### DISCUSSION TITLE
Joindre un .JPQ avec un message d'erreur concernant sa grosseur

### DISCUSSION THREAD

**Discussion by ChrisThe01** [MTESSDev/FRW#201] [Annonces]
Nous avons le message suivant lorsqu'un utilisateur joint un fichier de type JPG :
![image](https://github.com/MTESSDev/FRW/assets/129303869/432f4072-d292-41ce-bb04-435559f255e6)
Le type est 
      type: customfile
La validation est : 
     validations:                
                    mime: image/pjpeg,image/jpeg,application/vnd.openxmlformats-officedocument.wordprocessingml.document,application/vnd.oasis.opendocument.text,application/pdf,image/png,application/vnd.openxmlformats-officedocument.spreadsheetml.sheet

Comme l'utilisateur mentionne qu'il met des fichiers standards .PNG ou .JPG provenant de son téléphone et non d'internet. Un organisme va transmettre des documents dans ces formats lorsqu'il n'a pas de numériseur. Il va donc prendre des photos.

**Comment by j0ber** [comment]
**Tel qu'indiqué dans le message d'erreur** "La dimension du fichier doit être d'au moins 700px (largeur/hauteur)". 

Cela dit, **tel que décrit dans le guide de configuration d'un formulaire** ici : https://formulaires.it.mtess.gouv.qc.ca/fr/Form/7/P700U/0/N/#p=sectionAjouterPiecesJointes les dimensions permises sont paramétrables. 

Aucun téléphone ne prend des photos avec une résolution inférieure à 700px x 700px, et une image avec une résolution inférieure à cette taille risque d'être illisible ce qui risque de vous occasionner d'autres problèmes. À vous de voir.

Vous devez faire un test avec une image sur votre poste qui est trop petite c'est tout. Ce n'est pas représentatif de la réalité.

Contrairement à l'adage populaire LA TAILLE EST IMPORTANTE... 😉

**Comment by ChrisThe01** [comment]
Merci pour la réponse. Par contre, j'ai vu un comportement bizarre. Lorsqu'on a un fichier de type JPG avec une très bonne résolution, le fichier est quand même refusé. En regardant plus près, je constate que les meta données sont parfois absents sur un poste PC à un moment donné mais si copié sur un autre poste PC, ils peuvent redevenir présent !!!! Ce n'est pas un comportement très constante. Bizarre. Ceci dit, est-ce que on peut mettre la configuration d'où il n'y a aucune restriction minimale ?

**Comment by j0ber** [comment]
Pouvez-vous joindre votre fichier qui est rejeté s.v.p. ?

**Comment by ChrisThe01** [comment]
![TestJPG_Juju](https://github.com/MTESSDev/FRW/assets/129303869/53612dab-3aa9-4e8b-8b04-f3e606143d9b)

**Comment by j0ber** [comment]
Et quel est exactement le message d'erreur obtenu lorsque vous tentez d'ajouter cette image?

**Comment by ChrisThe01** [comment]
On le voit dans le message initiale.
![image](https://github.com/MTESSDev/FRW/assets/129303869/71874de6-f742-4a63-b8a5-6e2cfd4836fd)

**Comment by vinnyboii** [comment]
Bonjour,

L'image de votre chat fait 622 x 664 pixels ce qui explique le message d'erreur qui mentionne que la dimension minimale doit être 700 pixels. La vérification ne se fait pas sur la résolution en pixel/pouce, mais bien sur la largeur/hauteur de l'image en pixels.
![image](https://github.com/MTESSDev/FRW/assets/128411313/f9c77975-316a-4ff1-8d32-74f6dcd13a68)


Si vous voulez ajuster les spécifications minimales du contrôle de pièce jointe, référez vous à l'exemple 3 du P700 mentionné précédemment: https://formulaires.it.mtess.gouv.qc.ca/fr/Form/7/P700U/0/N/#p=sectionAjouterPiecesJointes 
![image](https://github.com/MTESSDev/FRW/assets/128411313/c2840fbe-397b-42a1-84f8-d16b4bd1b441)

Bonne journée
